[CLD-289]: fix(chains): migrate cldf.SolChain to cldf_solana.Chain - part 3

graham-chainlink opened the pull request:
`cldf_solana.Chain` is now the new import location, the `cldf.SolChain` will eventually be removed

JIRA: https://smartcontract-it.atlassian.net/browse/CLD-289

jkongie commented on deployment/ccip/changeset/solana/transfer_ccip_to_mcms_with_timelock_test.go:
```diff
@@ -115,18 +117,12 @@ func TestValidateContracts(t *testing.T) {
-		Bootstraps: 1,
-		Chains:     2,
-		SolChains:  1,
-		Nodes:      4,
```
Nice. This probably solves alot of flakiness, and saves on CI time

graham-chainlink committed:
fix(chains): migrate cldf.SolChain to cldf_solana.Chain - part 3

`cldf_solana.Chain` is now the new import location, the `cldf.SolChain` will eventually be removed

JIRA: https://smartcontract-it.atlassian.net/browse/CLD-289
